block-producer/src/block_builder/mod.rs

```diff
@@ -108,9 +117,15 @@ where
             produced_nullifiers,
         };
 
+        // TODO: Change to block.hash(), once it implemented
+        let block_hash = block.header.hash();
+
+        info!(target: COMPONENT, block_num, %block_hash, "built block");
+        debug!(target: COMPONENT, ?block);
+
         self.state_view.apply_block(block).await?;
 
-        info!(COMPONENT, "block #{block_num} built!");
+        info!(target: COMPONENT, block_num, %block_hash, "block is applied!");
```

Comment:
Sorry for changing this again, but let's update the messages as follows:

- On line 123: "block built"
- On line 128: "block committed"